queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
```

Comment:
- [ ] This query returns the names of all animals that its name ends in "mon". However, we are asked to find all animals `not just the names of the animals but everything about the animals` that its name ends in "mon". A good query would be `SELECT * FROM animals WHERE name LIKE '%mon;`